### ISSUE TITLE
Fast scrolling

### ISSUE THREAD

**Issue by Pampeers** [ToCraft/woodwalkers-mod#157]
### Describe your idea, include images if necessary

Add a slider for faster scrolling, there is even a special place on the right for this.

**Comment by ToCraft** [comment]
Do you mean in the menu of the ReMorphed mod?

**Comment by Pampeers** [comment]
Yeah, if you have a lot of mobs, then it takes a very long time to scroll down
